AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -22,6 +22,10 @@
 ##head[prefix~="medium-com:"] ~ body .js-openInAppButton
 ##head[prefix~="medium-com:"] ~ body .js-postActionsBarContent > .js-readNextInteractions
 !
+navitime.co.jp###smartapp-banner-area
+navitime.co.jp###footer-app-dl-frame
+navitime.co.jp##.footer-invite-app-section
+navitime.co.jp##.app-download-footer-frame
```

Comment:
```
navitime.co.jp##.footer-invite-app-section
navitime.co.jp##.app-download-footer-frame
```
Not reproduced.

Reply:
These elements are here. UA is mobile. 
URL: https://www.navitime.co.jp/transfer/pass/?from=top.topics
<details><summary>navitime.co.jp##.footer-invite-app-section</summary>

![](https://i.imgur.com/aUKaGNE.png)
</details><br/>

<details><summary>navitime.co.jp##.app-download-footer-frame</summary>

![](https://i.imgur.com/CqCPfwt.png)
</details><br/>